utility/accessdata.py

```diff
@@ -12,9 +12,16 @@ def __init__(self, data_path_name):
         self.access_path = ''
         self._read_setup()
         self._set_private_connection()
-
-        if data_path_name is not None:
-            self.data_path_name = data_path_name
+        self._data_path_eval(path=data_path_name)
+
+    def _data_path_eval(self, path):
+        if path is not None:
+            self.data_path_name = path
+            if os.path.isabs(self.data_path_name):
+                self.access_path = path
+                self.external_path = True
+            else:
+                self.external_path = False
             self._path_setup()
```

Comment:
Check if _path_setup() should be called only in the else branch.

Reply:
Done: [86f913b](https://github.com/gergopokol/renate-od/pull/228/commits/86f913bf1b983e7f68c2bb8e3e3ffa75b44274c9)